mod/groups/views/default/groups/css.php

```diff
@@ -63,3 +63,7 @@
 	background-color: #4690D6;
 	color: white;
 }
+
+.elgg-menu-invitationrequest {
+	float: right;
```

Comment:
I know. I thought I was too tired last night, but it seems it wasn't just
me... Do you have any clue what's up?
On Apr 12, 2015 12:34 PM, "Juho Jaakkola" <notifications@github.com> wrote:

> In mod/groups/views/default/groups/css.php
> <https://github.com/Elgg/Elgg/pull/8154#discussion_r28202406>:
>
> > @@ -63,3 +63,7 @@
> >  	background-color: #4690D6;
> >  	color: white;
> >  }
> > +
> > +.elgg-menu-invitationrequest {
> > +	float: right;
>
> This css is not taking effect.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/8154/files#r28202406>.
>